### ISSUE TITLE
I cannot get 55.3 mIoU of LoveDA dataset using D2LS

### ISSUE THREAD

**Issue by kuazhangxiaoai** [XavierJiezou/D2LS#20]
<img width="1134" height="109" alt="Image" src="https://github.com/user-attachments/assets/e91e0a8e-58c0-43d0-bc9c-d2b95f87564f" />

I just get 54.87 mIoU of LoveDA dataset using D2LS. I used constractive loss. Does the contractive decrease mIoU?
Thanks for your answers.

**Comment by XavierJiezou** [comment]
This is the score we obtained after submission on the Loveda online test platform.

<img width="2622" height="1378" alt="Image" src="https://github.com/user-attachments/assets/f20111e4-3a52-4809-bba4-f4f7488244c7" />

<img width="1804" height="1108" alt="Image" src="https://github.com/user-attachments/assets/bce90580-7884-4aca-8745-3030863c5845" />

You could double-check your training hyperparameters and training data settings. Note that like prior methods, we adopt Loveda train+val as our training set.
